awswrangler/_utils.py

```diff
@@ -409,16 +408,32 @@ def check_schema_changes(columns_types: Dict[str, str], table_input: Optional[Di
                 )
 
 
-def pylist_to_arrow(mapping: List[Dict[str, Any]]) -> pa.Table:
-    names = list(mapping[0].keys()) if mapping else []
+def table_refs_to_df(
+    tables: Union[List[pa.Table], List["ray.ObjectRef"]], kwargs: Dict[str, Any]  # type: ignore
+) -> pd.DataFrame:
+    """Build Pandas dataframe from list of PyArrow tables."""
+    if isinstance(tables[0], pa.Table):
+        return ensure_df_is_mutable(pa.concat_tables(tables, promote=True).to_pandas(**kwargs))
+    return _arrow_refs_to_df(arrow_refs=tables, kwargs=kwargs)  # type: ignore
+
+
+def list_to_arrow_table(
```

Comment:
Do you want to keep this version of the function with the schema handling? [this](https://github.com/awslabs/aws-data-wrangler/blob/release-3.0.0/awswrangler/_utils.py#L412) is just doing the minimum